releasenotes/notes/bump-centos8-stream-snapshots-2023-09-04-a473edfd3f3b2298.yaml

```diff
@@ -0,0 +1,11 @@
+---
+security:
+  - |
+    Bumps CentOS Stream 8 snapshots to include fixes for Zenbleed
+    (CVE-2023-20593), Downfall (CVE-2022-40982) and Inception (CVE-2023-20569).
+    It is recommended that you update your OS packages and reboot into the kernel
+    as soon as possible.
+upgrade:
```

Comment:
Need a note that you need to run host configure with dnf tag to pick up the exclude arg.

Reply:
> Need a note that you need to run host configure with dnf tag to pick up the exclude arg.

Good call!

Reply:
Done in https://github.com/stackhpc/stackhpc-kayobe-config/pull/612/commits/5e2ab5088391186a8fcc67d71174991ff71fb922